lib/router.js

```diff
@@ -45,18 +46,26 @@ router.get('/slashCommands/ci', (req, res) => {
 router.get('/slashCommands/ext', (req, res) => {
     const command = req.query && req.query.command;
     const text = req.query.text;
+    const update = req.query.update;
 
     if (command && command === '/ramal') {
-        const findExtension = text ?
-            extCommand.find.bind(extCommand, text) :
-            extCommand.find.bind(extCommand);
-
-        findExtension(out => {
-            res.send(out);
-        });
+        if (update && update === 'true' && config.extMode === 'SHEET') {
+            extCommand.update(text, out => {
```

Comment:
Why does `update` take a `text`? I think we should separete concerns here and make `update` do only what it says it does: `update` something. We can search for `text` after it's done updating. Would you agree?